crates/anvil/src/eth/fees.rs

```diff
@@ -81,6 +95,15 @@ impl FeeManager {
         }
     }
 
+    /// Calculates the current blob gas price
+    pub fn blob_gas_price(&self) -> U256 {
+        if self.is_eip4844() {
+            self.base_fee_per_blob_gas()
+        } else {
+            *self.gas_price.read()
+        }
```

Comment:
pre-eip 4844 there is no blob gas price, so this is not correct, i think

Reply:
Yep, this function is not needed at all, so will just remove